ci/Jenkinsfile-release

```diff
@@ -157,10 +179,10 @@ def prepareReleaseNotes(sparkMajorVersions, commons) {
             git fetch --no-tags
             git checkout master
             git pull
-            git checkout -b "master-changelog-${versionNoSparkSuffix}"
+            git checkout -b "master-changelog-${versionNoSparkSuffix}-\${LAST_COMMIT}"
```

Comment:
To be able to generate the release notes on the same machine, we need to have unique branch name